.github/ISSUE_TEMPLATE/novo_municipio.yaml

```diff
@@ -0,0 +1,71 @@
+name: Novo Município
+description: Sugerir um novo município para ser adicionado ao projeto
+title: "[Novo spider]: Municipio-Sigla Estado"
+labels: ["type:spider"]
+body:
+  - type: input
+    id: population
+    attributes:
+      label: Número de habitantes
+      description: Quantos habitantes tem o município?
+      placeholder: ex. 100 mil
+    validations:
+      required: false
```

Comment:
Sugestão: como é uma pergunta não obrigatória, acho melhor deixar mais ao fim; ou talvez tirar mesmo, já que não é uma informação crucial para desenvolver o raspador

Reply:
Aqui pensei nela mais para ajudar na priorização, mas se não for o caso melhor tirar mesmo.

Reply:
Eu não consegui ver como a issue fica depois de criada, mas ter essa informação seria útil se pudessemos filtrar pela população da cidade (para ajudar a priorizar qual spider começar a trabalhar), mas caso contrário, me parece mais uma "curiosidade" do que algo que realmente precisamos ter.

O fato de a pergunta existir (mesmo que não obrigatória), as vezes pode fazer alguém não continuar o preenchimento do formulário.

Reply:
Acho que a prioridade faz mais sentido pelo nosso lado de mantenedoras/gestão do projeto do que das pessoas interessadas em abrir a issue (as vezes, elas n se importam com o que é prioritário pra gente, só quer ter a própria cidade no QD, sabe?). Daí n acho que precisa ter essa informação na issue; poderiamos ter uma label como "prioridade", que nós ficamos responsáveis de dar quando uma nova issue chegar. Aí tb resolve a questão do filtro, permitindo que alguém veja todas as issues com essa label pra escolher onde começar

Reply:
Pergunta removida